RubeesEat/Model/InMemoryBillRepository.cs

```diff
@@ -21,7 +21,7 @@ public IReadOnlyList<Bill> GetAll()
 
     public Bill? GetById(Guid guid)
     {
-        return _bills[guid];
+        return !_bills.TryGetValue(guid, out var bill) ? null : bill;
```

Comment:
Negations make stuff harder to read so I would invert the condition. Personal preference but I would place the result values from the ternary operator (`... ? ... : ...`) on separate lines to promote readability even more:

```c#
return _bills.TryGetValue(guid, out var bill)
    ? bill
    : null;
```